packages/core/admin/admin/src/content-manager/pages/EditView/Header/index.js

```diff
@@ -204,6 +206,63 @@ const Header = ({
           />
         </Dialog>
       )}
+      {showPublishConfirmation && (
+        <Dialog
+          onClose={onPublishPromptDismissal}
+          title="Confirmation"
+          labelledBy="confirmation"
+          describedBy="confirm-description"
+          isOpen={showPublishConfirmation}
+        >
+          <DialogBody icon={<ExclamationMarkCircle />}>
+            <Stack spacing={2}>
+              <Flex justifyContent="center" style={{ textAlign: 'center' }}>
+                <Typography id="confirm-description">
+                  {draftCount}
+                  {formatMessage(
+                    {
+                      id: getTrad(`popUpwarning.warning.has-draft-relations.message`),
+                      defaultMessage:
+                        '<b>{count, plural, one { relation is} other { relations are}}</b> not published yet and might lead to unexpected behavior.',
+                    },
+                    {
+                      br: () => <br />,
+                      b: (chunks) => <Typography fontWeight="bold">{chunks}</Typography>,
+                      count: draftCount,
+                    }
+                  )}
+                </Typography>
+              </Flex>
+              <Flex justifyContent="center" style={{ textAlign: 'center' }}>
```

Comment:
@ronronscelestes Does the DS have a better way to express this? Should we add this to the `Typography` component?

Reply:
oh yes not the first time we encounter this issue, that would be great to add it to `Typography` props

<img width="604" alt="image" src="https://user-images.githubusercontent.com/71838159/194527175-b586dc69-9de0-466f-a475-94cef8ce24c5.png">

Reply:
Do you mean adding a `textAlign` prop to `Typography`?

Reply:
we'll try to release the DS before Wednesday anyway - maybe we could add this new prop before so it'll be released and usable quickly?

Reply:
@jhoward1994 Or you could add a TODO if you want to release this asap. I think it's not very important and better get it moving and refactor later :)

Thanks @ronronscelestes !

Reply:
thanks, yep I'll do that

Reply:
wdyt of at least replacing the inline style with 

```js
// TODO: replace with textAlign Typography props when available
const FlexTextAlign = styled(Flex)`
 text-align: 'center'
`;

...

<FlexTextAlign justifyContent="center"> ... </FlexTextAlign>
```

I think I would prefer `styled-components` customization than `inline` style 👀